_episodes/07-development-setup.md

```diff
@@ -302,17 +325,25 @@ To explore other tools, have a look at ESMValTool documentation on
 >>
 >> 1. ``docformatter``: it is mentioned that "files were modified by this hook".
 >> We run ``git diff`` to see the modifications.
->> The syntax ``"""`` at the end of docstring is moved by one line.
+>> The output includes the following:
+>> ~~~bash
+>> +in the form of the popular warming stripes figure by Ed Hawkins."""
+>> ~~~
+>> 
+>> The syntax ``"""`` at the end of docstring is moved by one line. 
+Shifting it to the next line should fix this error.
 >> 2. ``flake8``: the error message is about an unused local variable ``nx``.
```

Comment:
This `2.` appears as a `1.` on the website. Maybe you have to indent the lines after the first `1.`?